src/integration-tests/integration-test.py

```diff
@@ -114,7 +115,7 @@ class Integration(BaseModel):
             ContainerType.inputs: 2,
         },
         reboot_after_setup=True,
-        target_options=["-runs=10000000"],
+        fuzzing_target_options=["-runs=10000000"],
```

Comment:
It'd be good to have some other use of `target_options`, _ideally_ in a way that the target actually depends on. Something like an `--enable-crash` flag that gets passed on to the target. That'd require a new target to be written, so not for this PR.

Reply:
Yes, we were discussing earlier how the test doesn’t actually test that the `target_options` is used.